contracts/minter-suite/Minters/MinterSetPriceERC20/MinterSetPriceERC20V2.sol

```diff
@@ -214,6 +218,7 @@ contract MinterSetPriceERC20V2 is ReentrancyGuard, IFilteredMinterV0 {
         uint256 _projectId,
         uint256 _pricePerTokenInWei
     ) external onlyArtist(_projectId) {
+        require(_pricePerTokenInWei > 0, "Price may not be 0");
```

Comment:
question - is the thinking here that if price of zero is desired, minter should be set to `MinterSetPrice` instead of `MinterSetPriceERC20`?

Reply:
Yeah that's my thinking here–unless there was some other reason we could imagine someone wanting to do a zero-price mint but also specifically use the ERC20 minter?

Reply:
that logic SGTM! 🙏